api/migrations/versions/1c3fda18bad6_.py

```diff
@@ -0,0 +1,413 @@
+        self.application_close = application_close
+        self.review_open = review_open
+        self.review_close = review_close
+        self.selection_open = selection_open
+        self.selection_close = selection_close
+        self.offer_open = offer_open
+        self.offer_close = offer_close
+        self.registration_open = registration_open
+        self.registration_close = registration_close
+        self.event_type = event_type
+class ApplicationForm(Base):
+    __tablename__ = 'application_form'
+    __table_args__ = {'extend_existing': True}
+
+    id = db.Column(db.Integer(), primary_key=True)
+    event_id = db.Column(db.Integer(), db.ForeignKey('event.id'), nullable=False)
+    nominations = db.Column(db.Boolean(), nullable=False)
+    is_open = db.Column(db.Boolean(), nullable=False)
+    
+
+    def __init__(self, event_id, nominations, is_open):
+        self.event_id = event_id
+        self.nominations = nominations
+        self.is_open = is_open
+
+class Section(Base):
+    __tablename__ = 'section'
+    __table_args__ = {'extend_existing': True}
+
+    id = db.Column(db.Integer(), primary_key=True)
+    application_form_id = db.Column(db.Integer(), db.ForeignKey('application_form.id'), nullable=False)
+    name = db.Column(db.String(255), nullable=False)
+    description = db.Column(db.String(255), nullable=False)
+    order = db.Column(db.Integer(), nullable=False)
+    depends_on_question_id = db.Column(db.Integer(), db.ForeignKey('question.id', use_alter=True), nullable=True)
+    show_for_values = db.Column(db.JSON(), nullable=True)
+
+    def __init__(self, application_form_id, name, description, order):
+        self.application_form_id = application_form_id
+        self.name = name
+        self.description = description
+        self.order = order
+
+
+class Question(Base):
+    __tablename__ = 'question'
+    __table_args__ = {'extend_existing': True}
+
+    id = db.Column(db.Integer(), primary_key=True)
+    application_form_id = db.Column(db.Integer(), db.ForeignKey('application_form.id'), nullable=False)
+    section_id = db.Column(db.Integer(), db.ForeignKey('section.id'), nullable=False)
+    type = db.Column(db.String(), nullable=False)
+    description = db.Column(db.String(), nullable=True)
+    headline = db.Column(db.String(), nullable=False)
+    placeholder = db.Column(db.String(), nullable=True)
+    validation_regex = db.Column(db.String(), nullable=True)
+    order = db.Column(db.Integer(), nullable=False)
+    options = db.Column(db.JSON(), nullable=True)
+    is_required = db.Column(db.Boolean(), nullable=False)
+    depends_on_question_id = db.Column(db.Integer(), db.ForeignKey('question.id'), nullable=True)
+    show_for_values = db.Column(db.JSON(), nullable=True)
+    
+    def __init__(self, application_form_id, section_id, questionType, headline, placeholder, order, validation_regex, description = None,  is_required = True, options = None):
```

Comment:
`validation_text` seems to be missing? See the model [here](https://github.com/deep-learning-indaba/Baobab/blob/a33af3e5c21ede8787c9bdd77e69f347a9d35b5a/api/app/applicationModel/models.py#L40) - please add it back and set it on the questions that have validation below..